catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -166,4 +175,19 @@ private String createWkt(String[] lowerCornerPosition, String[] upperCornerPosit
         }
         }
     }
+
+    private String convertToEPSG4326(String wkt, String crs) {
+
+        try {
+            Geometry geometry = wktReader.read(wkt);
+            Geometry convertedGeometry = GeospatialUtil.transformToEPSG4326LonLatFormat(geometry, crs);
+            if (convertedGeometry != null) {
+                return wktWriter.write(convertedGeometry);
+            }
+        } catch (ParseException | GeoFormatException e) {
+            LOGGER.debug("Unable to convert {} from {} to EPSG:4326", wkt, crs, e);
```

Comment:
if we can't parse the WKT what are the consequences? will this blow up in the CatalogProvider? - seems like we are failing quietly here which could push the issue further down the call stack (and be more difficult to debug)